apps/ui/modules/reservation.ts

```diff
@@ -34,35 +33,41 @@ export const getDurationOptions = (
   if (!minReservationDuration || !maxReservationDuration || !intervalSeconds)
     return [];
 
-  const timeOptions = [];
+  const durationOptions: OptionType[] = [];
```

Comment:
Yeah, `OptionType` is problematic, but it's used _everywhere_. While I'm not a fan of coding more lines which have to be refactored later, I think making a quick-fix for it whenever encountered (just to avoid `OptionType`) is any better. Optimal solution would be to refactor away entire `OptionType` in one go.

Reply:
Yes but you changed the type of durationOptions so the refactor belongs here. Typescript checks break if there is old code left that assumes duration is `string`.

Reply:
👍